model-registration.md

```diff
@@ -33,50 +33,52 @@ There are two main activities associated with registering a model to Dojo:
 
-2. **Containerization and Execution**: next, modelers are directed to a containerization environment. In this environment, modelers teach Dojo how to run their models, how to set parameters, and provide metadata about model output files. The end result is the publication of a Docker image and associated metadata about the registered model.
+2. **Containerization and Execution**: next, modelers are directed to a containerization environment. Here, modelers teach Dojo how to run their models and how to set parameters. They can also provide metadata about model output files. The end result is the publication of a Docker image and associated metadata about the registered model.
 
 To get started, visit [https://dojo-modeling.com](https://dojo-modeling.com/). Please reach out to [dojo@jataware.com](mailto:dojo@jataware.com) for credentials or help with the application. 
 
 ## Model Metadata and Provenance
 
-To begin the process, Dojo captures metadata about each model and its maintainer. It's important to be as thorough as possible to ensure the end-user can understand at a high-level what each model does, how it does it, and what it produces.
+To begin the process, Dojo captures metadata about each model and its maintainer. It's important to be as thorough as possible to ensure that the end-user can understand at a high-level what each model does, how it does it, and what it produces.
 
-![Model Registration](imgs/model_registration.png)
+![Model Registration](imgs/registration-1-model-overview.png)
 
 Key definitions:
 
-- `Model Name`: The end-user will see this name when running your model. While spaces and upper/lower case are allowed, _**do not use special characters, to include parenthesis**_.
-- `Model Website`: This can be a link to your model repository or another website that you may maintain that provides additional context about your model.
-- `Model Family`: Model family refers to a group of models with similar characteristics. If you have several procedures based on the same underlying model family, you can link those models or model procedures here under a common family name of your choosing, such as `LPJmL`. If your model does not have a natural grouping with other models, you can name it under an appropriate category related to your model or repeat your model name.
-- `Model Description`: Your description here is the forward-facing documentation about your model that the end-user will see. Include as much information as possible to explain what your model does and what it produces. Include any notes that may be required to explain any model idiosyncrasies. If your model takes a long time to run, you may want to include an estimated run time.
+- `Model Name`: (required) The end-user will see this name when running your model. While spaces and upper/lower case are allowed, _**do not use special characters, including parentheses**_.
```

Comment:
Maybe there's a better markdowney way to stylize this (required), if you think we need it at all

Reply:
Looks good to me.